library/src/main/AndroidManifest.xml

```diff
@@ -6,7 +6,7 @@
 
         <provider
             android:name=".provider.TrayContentProvider"
-            android:authorities="@string/tray__authority"
+            android:authorities="${applicationId}.tray"
```

Comment:
Instant Run only breaks when the `AndroidManifest.xml` changes. This change doesn't break it